draft-ietf-scitt-architecture.md

```diff
@@ -614,103 +624,103 @@ The retrieved Receipt may be embedded in the corresponding COSE_Sign1 document i
 
-Similarly, the fact that an Issuer can be held accountable for its Transparent Claims does not on its own provide any mitigation or remediation mechanism in case one of these Claims turned out to be misleading or malicious---just that signed evidence will be available to support them.
+Similarly, the fact that an Issuer can be held accountable for its Transparent Statements does not on its own provide any mitigation or remediation mechanism in case one of these Transparent Statements turned out to be misleading or malicious---just that signed evidence will be available to support them.
 
-Issuers SHOULD ensure that the Statements in their Claims are correct and unambiguous, for example by avoiding ill-defined or ambiguous formats that may cause Verifiers to interpret the Claim as valid for some other purpose.
+Issuers SHOULD ensure that the Statement payloads in their Signed Statements are correct and unambiguous, for example by avoiding ill-defined or ambiguous formats that may cause Verifiers to interpret the Signed Statement as valid for some other purpose.
 
 Issuers and Transparency Services SHOULD carefully protect their private signing keys
-and avoid these keys for any purpose not described in this architecture. In case key re-use is unavoidable, they MUST NOT sign any other message that may be verified as an Envelope.
+and avoid these keys for any purpose not described in this architecture document. In cases where key re-use is unavoidable, keys MUST NOT sign any other message that may be verified as an Envelope as part of a Signed Statement.
 
 ## Threat Model
 
-We provide a generic threat model for SCITT, describing its residual security properties when some of its actors (identity providers, Issuers, TS, and Auditors) are corrupt or compromised.
+We provide a generic threat model for SCITT, describing its residual security properties when some of its actors (identity providers, Issuers, Transparency Services, and Auditors) are corrupt or compromised.
 
 This model may need to be refined to account for specific supply chains and use cases.
 
-### Claim authentication and transparency.
+### Signed Statement Authentication and Transparency.
 
-SCITT primarily supports evidence of Claim integrity, both from the Issuer (authentication) and from the TS (transparency). These guarantees are meant to hold for the long term, possibly decades.
+SCITT primarily supports checking of Signed Statement authenticity, both from the Issuer (authentication) and from the Transparency Service (transparency). These guarantees are meant to hold for the extensive periods of time, possibly decades.
 
-We conservatively suppose that some issuers and some TS will be corrupt.
+It can never be assumed that some Issuers and some Transparency Services will not be corrupt.
```

Comment:
This contradicts statements about dissimilar TSs interoperating or working together in a trusted way. The use of trust anchors to validate key credentials implies trust. All of this can be compromised or is subject to malicious or improper behavior. It seems impossible to achieve both objectives at the same time. Some aspects of this document is overstated, but it isn't clear which part.